thehive4py/endpoints/alert.py

```diff
@@ -16,8 +17,19 @@
 
 
 class AlertEndpoint(EndpointBase):
-    def create(self, alert: InputAlert) -> OutputAlert:
-        return self._session.make_request("POST", path="/api/v1/alert", json=alert)
+    def create(
+        self, alert: InputAlert, observables_path: Optional[Dict[str, str]] = None
+    ) -> OutputAlert:
+        if observables_path:
+            files: List[Union[Tuple[str, Tuple], Tuple[str, str]]] = [
+                (key, self._fileinfo_from_filepath(path))
+                for key, path in observables_path.items()
+            ]
+            files.insert(0, ("_json", jsonlib.dumps(alert)))
```

Comment:
`requests` offers the possibility to pass your from data in the `data` argument and your binary data in the `files` argument, and then when it prepares the request it will combine them in the end.
So there is no need to mutate the `files` list and insert the `_json`.

On the other hand `files` could be a dictionary as well, which is easier to manage then a tuple of str-tuple.

```python
if observables_path:
    files: Dict[str, Union[str, Tuple] = {
                key: self._fileinfo_from_filepath(path)
                for key, path in observables_path.items()
            }
    files["_json"] = jsonlib.dumps(alert)
```